first-react-app/src/components/formComponents/FormCard.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import note from "../../assets/note.png";
+
+function FormCard(props: {
```

Comment:
you should use a named interface for component props.
you should use arrow function instead of regular function.
```suggestion
const FormCard = (props: FormCardProps) => {
```